packages/osbx/osbx.1.0.0/opam

```diff
@@ -11,7 +11,7 @@ depends: [
   "jbuilder" {build}
   "stdint" {build}
   "nocrypto" {build}
-  "angstrom" {build & < "0.7.0"}
+  "angstrom" {build & >= "0.2.0" & < "0.7.0"}
```

Comment:
@darrenldl It looks like seqbox needs a lower-bound constraint to ensure that the `BE` module is available.

Reply:
@seliopou Many thanks for keeping everyone updated!